.github/actions/deploy-sdk-docker-compose/action.yml

```diff
@@ -164,6 +164,7 @@ runs:
           export OWSUB_SYSTEM_URI_PUBLIC="https://${{ steps.set_outputs.outputs.gateway_url }}:16006";
           export OWRRM_SERVICECONFIG_PRIVATEENDPOINT="http://owrrm.wlan.local:17007";
           export OWRRM_SERVICECONFIG_PUBLICENDPOINT="http://${{ steps.set_outputs.outputs.gateway_url }}:16789";
+          export CERTIFICATES_ALLOWMISMATCH="false";
```

Comment:
If I understood this new config option correctly I think this would break the Docker Compose deployment workflows since we're using the same AP cert for all workflow runs. So in this case we would need to set it to `true` because otherwise we would have to generate a cert for every run.

Reply:
Good catch. Brought that up to Jaspreet in I totally agree with you. Brought that up to Jaspreet in https://telecominfraproject.atlassian.net/browse/WIFI-11905

Reply:
Allowed this while disallowing this in other cases